app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -578,23 +742,22 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         // CAUTION set item properties programmatically otherwise it would not be accepted by
         // appcompat itemsinflater.inflate(R.menu.videoitem_detail, menu);
 
-        inflater.inflate(R.menu.video_detail_menu, menu);
+        /*inflater.inflate(R.menu.video_detail_menu, menu);
```

Comment:
Remove this override, `updateMenuItemVisibility`, `onOptionsItemSelected`, `showInstallKoreDialog`, `setupActionBar`, since there is no toolbar menu anymore